package.json

```diff
@@ -90,6 +90,8 @@
 	},
 	"devDependencies": {
 		"@eightshift/storybook": "^5.6",
+		"@jest/globals": "^26.6.2",
+		"babel-preset-env": "^1.7.0",
```

Comment:
we are using @wordpress/default babel setup and that setup incorporates babel-preset-env 
https://github.com/WordPress/gutenberg/blob/a9ed7ced5e10ec5ff54d792ac18150cd1c5139b8/packages/babel-preset-default/index.js#L36

So se don't need to implement this. Also this is an older version because Bable has changed the name so all the packages starts with @babel/